pkg/sql/execinfrapb/processors_bulk_io.proto

```diff
@@ -292,6 +292,8 @@ message RestoreFileSpec {
   optional string path = 2 [(gogoproto.nullable) = false];
   reserved 3;
   reserved 4;
+  optional roachpb.Span span = 5 [(gogoproto.nullable) = false];
```

Comment:
I wonder if we should call this... something more specific? If I am reading it correctly, this is the Backup MfstFile Span, which is not equal to, or even a subset to, the restore entry span, so it seems confusing to have it around as "Span". 

Maybe we should  intersect it with the entry span before we put the spec in the entry, and then we just call it SubSpan?